app_user/templates/app_user/detail.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html' %}
+
+{% load template_extras %}
+{% load staticfiles %}
+
+{% block body_classes %}user-detail{% endblock %}
+
+{% block content %}
+    <div class="container">
+        <h1 class="line">
+            <span class="grey-text title-type">Nutzer</span><br />
+            <span class="orange-text text-lighten-1">{{ user.pseudonym_or_full_name }}</span>
+        </h1>
+
+        <div class="row">
+            <div class="col s4">
+                <img src="{% static "img/blank-user-profile.png" %}" class="user-profile" />
+            </div>
+            <div class="col s8">
+                <p>Name: <strong>{{ user.full_name }}</strong></p>
+                <p>Bewertung:
+                    <strong>
+                        <span class="seller-rating">
+                          <i class="material-icons seller-rating-star">star</i>
+                          <i class="material-icons seller-rating-star">star</i>
+                          <i class="material-icons seller-rating-star">star</i>
+                          <i class="material-icons seller-rating-star">star_half</i>
+                          <i class="material-icons seller-rating-star">star_border</i>
+                        </span>
+                    </strong>
+                </p>
+                <p>Adresse: <strong>Musterstr. 3, 12047 Berlin</strong></p>
```

Comment:
Steht nicht in den Anforderungen und ist somit erst einmal Adresse & Ort aus dem Template geflogen. Wir sollten gucken ob wir das Usermodel auch noch falls nötig anpassen.